plugin/evm/vm.go

```diff
@@ -693,6 +669,24 @@ func (vm *VM) getAtomicTx(txID ids.ID) (*Tx, uint64, error) {
 	return tx, height, nil
 }
 
+// getAtomicTx returns the requested transaction, status, height if accepted,
+// and whether or not it was found.
+func (vm *VM) getAtomicTx(txID ids.ID) (*Tx, Status, uint64, bool) {
```

Comment:
This should really be returning an error (Check for `database.NotFound` vs other errors)